wandb/sdk/internal/system/assets/network.py

```diff
@@ -68,6 +68,71 @@ def aggregate(self) -> dict:
         return {self.name: aggregate}
 
 
+class NetworkTrafficSent:
+    """Network traffic sent."""
+
+    name = "network.traffic_sent"
```

Comment:
What do you want to call it? upload_speed (as in go) or traffic_sent? (probably the former?)
```suggestion
    name = "network.upload_speed"
```